romancal/regtest/test_mos_pipeline.py

```diff
@@ -32,123 +27,93 @@ def test_level3_mos_pipeline(rtdata, ignore_asdf_paths):
-        "DMS400 MSG: Testing completion of skymatch in the Level 3  output......."
-        + passfail(model.meta.cal_step.skymatch == "COMPLETE")
-    )
-    assert model.meta.cal_step.skymatch == "COMPLETE"
-    pipeline.log.info(
-        "Status of the step:             skymatch    "
-        + str(model.meta.cal_step.skymatch)
-    )
-    pipeline.log.info(
-        "DMS400 MSG: SkyMatchStep added meta.background? :"
-        f'  {hasattr(model.meta.individual_image_meta, "background")}'
-    )
-    assert hasattr(model.meta.individual_image_meta, "background")
-
-    pipeline.log.info(
-        "DMS400 MSG: SkyMatchStep populated meta.background.level? :"
-        f"  {any(model.meta.individual_image_meta.background['level'] != 0)}"
-    )
-    assert any(model.meta.individual_image_meta.background["level"] != 0)
-    pipeline.log.info(
-        "DMS86 MSG: Testing completion of outlier detection in the Level 3 image output......."
-        + passfail(model.meta.cal_step.outlier_detection == "COMPLETE")
-    )
-    assert model.meta.cal_step.outlier_detection == "COMPLETE"
-    pipeline.log.info(
-        "Status of the step:             outlier_detection    "
-        + str(model.meta.cal_step.outlier_detection)
-    )
-    assert model.meta.cal_step.resample == "COMPLETE"
-    pipeline.log.info(
-        "Status of the step:             resample          "
-        + str(model.meta.cal_step.resample)
-    )
-    pipeline.log.info(
-        "DMS86 MSG: Testing completion of resample in the Level 3 image output......."
-        + passfail(model.meta.cal_step.resample == "COMPLETE")
-    )
-
-
-@pytest.mark.bigdata
-@pytest.mark.soctests
-def test_hlp_mosaic_pipeline(rtdata, ignore_asdf_paths):
-    """Tests for level 3 mosaic requirements DMS373"""
-    rtdata.get_asn("WFI/image/L3_mosaic_asn.json")
 
-    # Test Pipeline
-    output = "r0099101001001001001_r274dp63x31y81_prompt_F158_i2d.asdf"
-    rtdata.output = output
-    args = [
-        "roman_mos",
-        rtdata.input,
-    ]
-    MosaicPipeline.from_cmdline(args)
-    rtdata.get_truth(f"truth/WFI/image/{output}")
-    pipeline = MosaicPipeline()
-    diff = compare_asdf(rtdata.output, rtdata.truth, **ignore_asdf_paths)
+def test_output_matches_truth(output_filename, truth_filename, ignore_asdf_paths):
+    # DMS356
+    diff = compare_asdf(output_filename, truth_filename, **ignore_asdf_paths)
     assert diff.identical, diff.report()
 
-    model = rdm.open(rtdata.output, lazy_load=False)
 
-    pipeline.log.info(
-        "DMS373 MSG: Testing the creation of a Level 3 mosaic image resampled to a skycell"
-        + passfail(model.meta.cal_step.resample == "COMPLETE")
-    )
+def test_thumbnail_exists(thumbnail_filename):
+    # DMS356
+    # FIXME was not an assert before
+    assert os.path.isfile(thumbnail_filename)
+
+
+def test_preview_exists(preview_filename):
+    # DMS356
+    # FIXME was not an assert before
+    assert os.path.isfile(preview_filename)
```

Comment:
This was not previously as assert. It passes so I'm assuming it's ok to make it one.